src/components/image-container/image-container.scss

```diff
@@ -19,6 +19,23 @@
 	border: 0;
 	height: auto;
 	margin: 0;
+	object-fit: cover;
```

Comment:
@JosephCurley -- let's discuss this on a call.  Not sure we want `image-container` to have this opinion without a modifier like `image-container--cover-center` ?